Remove deprecation warning of new instance()

kabutz opened the pull request:
Improved: Removed deprecation warning

Explanation: Class.newInstance() has a flaw, in which any Throwable from the constructor is thrown unchecked. It was deprecated in Java 9, but not marked for removal. It shows as a deprecation warning when compiling OfBiz with Java 11.

Thanks: Heinz Kabutz

kabutz committed:
Removed direct call to Class.newInstance()

mbrohl commented:
Hello @kabutz ,

thanks for your contribution!

Can you please rebase your feature branch instead of merging it to avoid the additional merge commit? Thanks!

kabutz commented:
> Hello @kabutz ,
> 
> thanks for your contribution!
> 
> Can you please rebase your feature branch instead of merging it to avoid the additional merge commit? Thanks!

Of course.

JacquesLeRoux commented:
Hi @kabutz,

Also please update the "commit comment" (ie the PR comment). I have already cleaned it up a bit, but it misses an explanation (eg: in what env did you cross warning/s?, etc.) and it would be better if a Jira could be created and its number put into the (OFBIZ-...) part

TIA

kabutz commented:
> Hi @kabutz,
> 
> Also please update the "commit comment" (ie the PR comment). I have already cleaned it up a bit, but it misses an explanation (eg: in what env did you cross warning/s?, etc.) and it would be better if a Jira could be created and its number put into the (OFBIZ-...) part
> 
> TIA

Thanks @JacquesLeRoux - I've added an explanation.
